src/lib/mdns/tests/TestServiceNaming.cpp

```diff
@@ -81,9 +81,82 @@ void TestExtractIdFromInstanceName(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, ExtractIdFromInstanceName("1234-", &peerId) != CHIP_NO_ERROR);
 }
 
+void TestMakeServiceNameSubtype(nlTestSuite * inSuite, void * inContext)
+{
+    // TODO(cecille): These need to be changed to remove leading zeros
+    static constexpr size_t kSize = 16;
```

Comment:
static here is not needed, I don't think

Reply:
gone.